operations/finance/operations-playbook.md

```diff
@@ -1,2 +1,242 @@
+|[Payments via Accrualify](https://docs.google.com/document/d/1HOUvNFWpCEznwEyJmWoC5aqfJaIDK3FfYVzUt31MQp8/edit)|`Accrualify` `Netsuite`|Playbook| |Weekly|Jeff Dynda|Accounting|
+|[Create wire via Transferwise](https://docs.google.com/document/d/1faaLx4aGwjGIW7EPdlwOAu1EBdG_NLGdagq9KzzzPMc/edit)|`Transferwise` `Netsuite`|Playbook| |Weekly|Jeff Dynda|Accounting|
+|[Pay ROW staff via Transferwise](https://docs.google.com/document/d/1_vEICaFz3w4QAK-_j3F85noVll4Q8X7I80GJGFjAKes/edit)|`Transferwise` `Netsuite`|Playbook| |Weekly|Jeff Dynda|Accounting|
+|[Settel funding between subsidiaries](https://docs.google.com/document/d/1qLHetaaljiJTF4fWb56OcnIR5XLRFx7kRS3TvNwf9yM/edit?usp=sharing)|`Netsuite`|Playbook| |Monthly|Ben Graham|Accounting|
+|[Divvy virtual credit card walkthrough](https://docs.google.com/document/d/1Yypp0jbciDspLDp1_bx0aKi7EuCLs9I6ppc2-zGm6zM/edit#heading=h.hsezwk8fw18o)|`Divvy` `Netsuite`|Process Narrative| |Weekly|Ben Graham|Accounting|
+|[Issue, book, monitor, and reconcile divvy corporate cards](https://docs.google.com/document/d/1kLrTfBqeWyPyXia17LJwLn-vI31j2uMJbs_DEd46p7E/edit#heading=h.t8lxnw6fufgs)|`Divvy` `Netsuite` `Accrualify`|Playbook| |Weekly|Ben Graham|Accounting|
+|[Reconciling cash](https://docs.google.com/document/d/1igoLD-tSKXqukAcQkanjHDz2FJGp_CT9pOIlaovn3kM/edit#heading=h.3gt7174pkqjt)|`Netsuite`|Playbook| |Weekly|Jeff Dynda|Accounting|
+|[Manage and reconcile Paypal](https://docs.google.com/document/d/1FibtBTryBYWSxKEoLULnXbGNK8NjiUjBHhzdXZPQAeM/edit#heading=h.qz5k1zqf0i4s)|`Netsuite` `Paypal`|Playbook| |Weekly|Jeff Dynda|Accounting|
+|[Cash disbursement waterfall](https://docs.google.com/document/d/1hpD-c16kx_lliXhuxkF9oNZED9idlzk2O-0nJQlptbY/edit#)|`Netsuite`|Playbook| |Monthly|Jeff Dynda|Accounting|
+
+
+### Revenue and acounts receivable
+Internal processes and framework for order processinng, sales order review and invoicing, recognizing US GAAP revenue, cash collections, and uncollectible receivables.
+
+|Process Description/Doc Link|System/Tool|Document Type|Flow Chart|Occurrence|AOR|Access Rights|
+|:----|:----|:----|:----|:----|:----|:----|
+|[Lead to contract walkthrough](https://docs.google.com/document/d/1DucB1CJqKqucLTw_9eqsjm6Fo7mE0J1vtaYcP_2Q_as/edit)|`SFDC`|Process Narrative| |Daily|Ben Graham|Mattermost|
+|[Order management training videos](https://drive.google.com/drive/folders/1fgpSWCvJ0G97iSngvQr4UlTrv6xatzu7?usp=sharing)|`SFDC`|Drive| |Daily|Nikki Johnson, Jeff Dynda|Accounting|
+|[Order management resources](https://drive.google.com/drive/folders/1qYOLgNIFPXxE4Kl875OJWYjv5wHMxlpj?usp=sharing)|`SFDC`|Drive| |Daily|Nikki Johnson, Jeff Dynda|Accounting|
+|[Non-production key license provisioning](https://docs.google.com/document/d/1eSVIB40SpEaTHbjd2E9Sl8jt9j0U_Ngjy-aXnQsvbIY/edit?usp=sharing)|`Netsuite`|Playbook| |As needed|Jeff Dynda|Accounting|
+|[True-up customer seats]|`SFDC`|Playbook| |Quarterly|Jeff Dynda|Accounting|
```

Comment:
@TQuock Is there a link to true-up customer seats you'd like to include here?

Reply:
Thanks for highlighting, Jason! These are placeholders for now as we finalize the documentation. More PRs to come in the next few weeks.